src/constitution.md

```diff
@@ -0,0 +1,192 @@
+
+# Membership
+
+Membership is open to all students, faculty, and staff at the University of Guelph School of Computer Science.
+This includes undergraduate students and graduate students of computing-related degrees.
+
+# Governance
+
+All club activities are governed by an appointed executive body, as well as one or more committees.
+
+## Executive
+
+The Executive body shall consist of the following roles and their responsibilities.
+
+  i. **President**
+
+     Responsibilities:
+
+      - Lead the Executive, delegating administrative tasks as necessary
+      - Approve proposed budgets for each semester
+      - Ensure the success of the Club in accordance with its goals
+      - Provide guidance to Club Committees
+      - Ensure that new Club Committee Members are trained on club policies and procedures
+      - Maintain the GCC GitHub Organization
+      - Liaise with SOCIS as required by SOCSSO Policies
+      - Ensure that GCC has a faculty advisor for each semester of operation
+      - Ensure that GCC qualifies for accreditation each semester, as a SOCSSO Club
+      - Ensure that club does not fall into probationary status, as defined by SOCSSO Policies
+
+ ii. **Treasurer**
+
+     Responsibilities:
+
+      - Draft and share financial budgets for each semester of operation
+      - Attend CEPSSC and SOCIS meetings to PDR for funds
+      - Ensure that all expenses (snacks, honorariums, prizes, etc.) are purchased, accounted for in the budget, and reimbursed to the purchaser
+      - Create and present a summary of income and expenditures each semester
+
+
+iii. **Director of Sponsorship**
+
+     Responsibilities:
+
+      - Draft and maintain a sponsorship plan for external organizations which are interested in funding club operations
+      - Correspond with industry representatives
+      - Invite industry representatives to campus to provide guest lectures and workshops
+      - Collaborate with the School of Computer Science Co-op Coordinators to maintain industry interest in Guelph talent
+
+ iv. **Director of Advertising and Outreach**
+
+     Responsibilities:
+
+      - Visit classrooms at the beginning of each semester to promote the Club
+      - Create attractive, informative digital and print advertising for each GCC event
+      - Create and share a plan for advertising, including an outline of different platforms which will be used and any costs that may be incurred by these platforms
+      - Collaborate with SOCIS to ensure access to social media advertising outlets, such as GryphSlack and Instagram
+
+  v. **Director of Events**
+
+     Responsibilities:
+
+      - Coordinate with weekly presenters to review and polish presentation and workshop materials
+      - Host office hours in the Club Space for students who wish to improve their presentation skills
+
+ vi. **Executive at Large**
```

Comment:
How many Executive at Large's are there? There are currently 2, must there be 2 or any number?

Reply:
Usually executive at large is a nebulous term used to mean "generic board member" and the number isn't defined.
I think as long as we list all the current execs at large, we should be fine. It gives us the room to have as big of an exec team as we want if more people come up that are willing to participate.

Reply:
@StealthGiraffe has it exactly right, if it isn't clear, there can be multiple chairs and exec at large. I can clarify this though.

Reply:
Added a note to this role that clarifies that there can be multiple exec.